lib/web_ui/lib/src/engine/initialization.dart

```diff
@@ -229,6 +229,9 @@ Future<void> initializeEngineUi() async {
   RawKeyboard.initialize(onMacOs: ui_web.browser.operatingSystem == ui_web.OperatingSystem.macOs);
   KeyboardBinding.initInstance();
 
+  // Ensures Flutter renders a global "generator" meta-tag.
+  ensureMetaTag('generator', 'Flutter');
```

Comment:
Can we do this in `flutter.js` so it's loaded asap?

Reply:
Oh, just saw your comment:

> Added it here, instead of flutter.js so we can have a proper test for the feature.

I think we should add it to `flutter.js` if it's the more appropriate place for it. For testing, I think we have a couple of flutter.js tests already that we can steal ideas from. Alternatively, we can write an integration test on the flutter/flutter side. We're merging repos soon anyway, so the disadvantages of cross-repo testing will go away.

Reply:
I agree that `flutter.js` has the advantage of injecting the meta tag earlier, but I would like us to keep in mind that `flutter.js` is optional (is this not the case anymore?). So we still need to `ensureMetaTag` here to guarantee that the tag is always added.

If we want to add the meta tag in the absolute earliest moment, we could add it in `index.html`. But again, we still need to do it in `initializeEngineUi` as a backup.